service.subtitles.subtitle/service.py

```diff
@@ -125,11 +125,13 @@ def takeTitleFromFocusedItem():
     labelTVShowTitle = xbmc.getInfoLabel("ListItem.TVShowTitle")
     labelSeason = xbmc.getInfoLabel("ListItem.Season")
     labelEpisode = xbmc.getInfoLabel("ListItem.Episode")
+    isItMovie = xbmc.getCondVisibility("Container.Content(movies)") or labelType == 'movie'
```

Comment:
Ok,
I`ve reviewed the code again, and the code not affects regular usage of subtitle addon.
It only solves specific scenario for users that want to check if there is a subtitle from context menu.